### PR TITLE
add border style in style.css

### PR DESCRIPTION
**Pull request by programadopormartin**
Add a purple border for body element

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,5 +1,6 @@
 body {
     background-color: aquamarine;
+    border: solid violet thick;
```
Comment (rubenvidales) [inline style.css]: Is not violet, should be pink

**Comment by programadopormartin** [comment]
Color solved

File: style.css (hunk 1 above)
Comment (programadopormartin) [reply style.css]: color solved

File: style.css [hunk 2]
```diff
@@ -1,5 +1,6 @@
 body {
     background-color: aquamarine;
+    border: solid pink thick;
 }
 
 h1 {
```
Comment (programadopormartin) [inline style.css]: ok

File: style.css [hunk 3]
```diff
@@ -1,5 +1,6 @@
 body {
     background-color: aquamarine;
+    border: solid pink thick;
```
Comment (rubenvidales) [inline style.css]: 👍

**Pull request merged**